code/eric/HTML & CSS/lab_03.html

```diff
@@ -0,0 +1,152 @@
+    </head>
+
+<style>
+    h1, h2, p {
+        text-align: center;
+    }
+    body {
+        background-color: rgba(107, 133, 155, 0.449);
+    }
+    div {
+        width: 80%;
+        margin-left: auto;
+        margin-right: auto;
+        overflow: scroll;
+    }
+    .box {
+        border-style: solid;
+        display: flex;
+        align-items: flex-start;
+        flex-direction: row;
+        background-color: white;
+        overflow: scroll;
+        padding: 20px;
+    }
+    .topnav a {
+        float: left;
+        color: #f2f2f2;
+        text-align: center;
+        padding: 14px 16px;
+        text-decoration: none;
+        font-size: 17px;
+    }
+    .topnav a:hover {
+        background-color: #ddd;
+        color: black;
+    }
+    .topnav a.active {
+        background-color: black;
+        color: white;
+    }
+    .sidenav {
+        width: 80px;
+        position: fixed;
+        z-index: 1;
+        top: 200px;
+        left: 15px;
+        background: #eee;
+        overflow-x: hidden;
+        padding: 8px 0;
+    }
+    .sidenav a {
+        padding: 6px 8px 6px 20px;
+        text-decoration: none;
+        font-size: 15px;
+        color: #2196F3;
+        display: block;
+    }
+    .sidenav a:hover {
+        color: #064579;
+    }
+    .footer {
+        position: fixed;
+        left: 0;
+        bottom: 0;
+        width: 100%;
+        height: auto;
+        background-color: rgba(107, 133, 155, 0.449);
+        color: white;
+        text-align: left;
+    }
+</style>
+
+<header>
+<div class="topnav">
+    <a class="active" href="#home">Home</a>
+    <a href="#news">News</a>
+    <a href="#contact">Contact</a>
+    <a href="#about">About</a>
+</div>
+</header>
+
+<aside>
+<div class="sidenav">
+    <a href="#home">Home</a>
+    <a href="#news">News</a>
+</div>
+</aside>
+
+<h1>Blog Stories</h1>
+    <div class="box">
+        <div>
+    <section><article>
+    <h2>Ipsum 1</h2>
+    <img src="https://via.placeholder.com/150" width="300" height="200">
+    <p>In software, we rarely have meaningful requirements. Even if we do, the only measure of success that matters is whether
+    our solution solves the customer's shifting idea of what their problem is. (Jeff Atwood) But what is it good for?
+    (Engineer at the Advanced Computing Systems Division of IBM, commenting on the microchip, 1968) The most amazing
+    achievement of the computer software industry is its continuing cancellation of the steady and staggering gains made by
+    the computer hardware industry. (Henry Petroski) There are two ways to write error-free programs; only the third one
+    works. (Alan J. Perlis) There are two ways of constructing a software design. One way is to make it so simple that there
+    are obviously no deficiencies. And the other way is to make it so complicated that there are no obvious deficiencies.
+    (C.A.R. Hoare).</p>
+    </section></article>
```

Comment:
if you want to have an `<article>` inside of a `<section>` the format should look something like 
```
<section>
   <article>
      the content you want to display
   </article>
</section>
```